### PR TITLE
chore: add 'scheduleMessages' option

### PR DESCRIPTION
**Pull request by Bounteous17**
- Update node version 16
- Tests are no longer broken
- The topic messages consumption can be scheduled now

### CODE REVIEW

**Review by kevinccbsg — APPROVED** [review]
It covers [what we need](https://medium.com/event-driven-utopia/azure-service-bus-essentials-scheduled-messages-98fd542f9d55) @Bounteous17 so it looks good to me

File: test/topics/e2e.test.js [hunk 1]
```diff
@@ -75,6 +75,27 @@ describe('Topics - Systemic Azure Bus API', () => {
...
+			if (!properties.messageId || +properties.messageId !== _messageId) return;
+			expect((Date.now() - startTimestamp) / 1000).to.be.greaterThan(5);
+			expect(msg).to.have.keys('body', 'properties', 'applicationProperties');
+			resolve();
+		};
+		busApi.safeSubscribe('assess', handler(+messageId));
+		const scheduledEnqueueTimeUtc = new Date(Date.now() + 5000);
+		// eslint-disable-next-line prefer-const
+		startTimestamp = Date.now();
+		await publish(payload, { messageId, scheduledEnqueueTimeUtc });
```
[the remark this answers is not in the document]
Comment (Betisman) [reply test/topics/e2e.test.js:96]: Why do you propose to remove the messageId?

File: test/topics/health.test.js [hunk 2]
```diff
@@ -29,10 +29,11 @@ describe('Health check', () => {
 		before(async () => {
 			busApi = await bus.start({ config });
 			await busApi.purgeDlqBySubcriptionId('assess');
+			await bus.stop();
 		});
 
-		after(async () => {
-			await bus.stop();
+		beforeEach(async () => {
+			busApi = await bus.start({ config });
```
Comment (Betisman) [inline test/topics/health.test.js:36]: Aren't we missing the `after`?
```
after(async () => {
			await bus.stop();
});
```

Comment (Bounteous17) [reply test/topics/health.test.js:36]: https://github.com/guidesmiths/systemic-azure-bus/pull/54/commits/f75a47bbc4b4c1b0bdcbba9ef1fc2fa7937d37f2

File: test/topics/e2e.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bounteous17) [reply test/topics/e2e.test.js:96]: `messageId` is an optional property, let's leave it as it is :smiley_cat:

**Pull request merged**